Updates to entire draft based on Anoop's G-DOC

smishra1200 opened the pull request:
Updated entire draft to add all changes from Anoop's G-Doc and general clean up.

smishra1200 committed:
Updates to entire draft based on Anoop's G-DOC

Updated entire draft to add all changes from Anoop's G-Doc and general clean up.

zaheduzzaman commented on draft-mishra-scone-usecase.md:
```diff
@@ -67,142 +67,161 @@ informative:
+A media delivery technique that dynamically adjusts the quality of content (e.g., video) in real-time based on the
+available network bandwidth.
+
+- Application End-point
```
We are defining Application "End-point" but using Application "endpoint". If we are defining "end-point" then we need change where we use "endpoint".

smishra1200 replied:
Good catch Zahed. I should have used application endpoint. Correcting it  now on the submitted patch

zaheduzzaman commented on draft-mishra-scone-usecase.md:
```diff
@@ -216,52 +235,45 @@ application flows. Each session is bound to an IP address and port,
+mechanisms. In certain deployments, network operators may choose to apply distinct
+QoS policies to SCONE-enabled flows. However, SCONE enabled flows are typically
+used for adaptive bit-rate applications and are expected to be treated as
+default QoS flow for the corresponding PDU session.
```
why are we useing the default QoS here? We need to define default QoS then. To my knowledge other then 3GPP default QoS does not exists. the same applies for "PDU session".

smishra1200 replied:
Need Anoop to weigh in as the text came from him

zaheduzzaman replied:
The text was what we agreed after the meeting we had. and we had not action point to change or update the text in this section. Anoop was good with the text. I insist not to change it now.

anooptomar29 replied:
Zahed and I discussed and we agreed to replace differentiated QoS by advised bit rate

zaheduzzaman commented on draft-mishra-scone-usecase.md:
```diff
@@ -272,56 +284,59 @@ SCONE signaling can be integrated into existing operational and
+
+Similarly, in 5G mobile SCONE packets are sent by the Network Element (i.e., the UPF) over the
+corresponding PDU session (N3 Interface + Air Interface) that is carrying the IP flow for which
+the Network element intends to send the bit-rate.
```
Why are we introducing so much 3GPP details here again? I would simply remove the examples, if we want to change anything.

smishra1200 replied:
That was a suggestion from Anoop but if you want me to remove, I can

zaheduzzaman replied:
We can move such details to specific 5G/4G sections.

smishra1200 committed:
Fix Application End-point to Application endpoint per Zahed

1) Fix Application End-point to Application endpoint per Zahed
2) Remove "However, due to the high flexibility of the 5G's
SBA, any network component capable of meeting the defined Applicability and Manageability considerations may act as a SCONE-capable Network Element" Per Anoop

zaheduzzaman committed:
Fixed Zahed's comment
